examples/arithmetic/tests/bindings/test_arithmetic.kts

```diff
@@ -1,6 +1,20 @@
 import uniffi.arithmetic.*;
 
-assert(add(2, 3, Overflow.SATURATING) == 5L)
+assert(add(2, 4) == 6L)
+assert(add(4, 8) == 12L)
 
+try {
+    sub(0, 2)
+    throw RuntimeException("Should have thrown a IntegerOverflow exception!")
+} catch (e: ArithmeticErrorException) {
```

Comment:
In swift and python we explicitly test that this was the `IntegerOverflow` variant, but here we test only that it was an `ArithmeticError`. Does it work if we catch `ArithmeticErrorException.IntegerOverflow` here like the other languages do?

(I note that the todolist example has a similar problem, but we don't need to fix that here)